realm/realm-library/build.gradle

```diff
@@ -37,6 +37,7 @@ repositories {
 
 dependencies {
     provided 'io.reactivex:rxjava:1.1.0'
+    provided 'junit:junit:4.12'
```

Comment:
`RealmRobolectricRule` needs this. Users can use `RealmRobolectricRule` as following:
```
public class RobolectricTest {
    ...
    @Rule
    public RealmRobolectricRule realmRobolectricRule = new RealmRobolectricRule();
    ...
}
 ```